common/test/acceptance/pages/lms/course_wiki.py

```diff
@@ -0,0 +1,81 @@
+"""
+Wiki tab on courses
+"""
+
+from .course_page import CoursePage
+from ...pages.studio.utils import type_in_codemirror
+
+
+class CourseWikiPage(CoursePage):
+    """
+    Course wiki navigation and objects.
+    """
+
+    url_path = "wiki"
+
+    def is_browser_on_page(self):
+        """
+        Browser is on the wiki page if the wiki breadcrumb is present
+        """
+        return self.q(css='.breadcrumb').present
+
+    def open_editor(self):
+        """
+        Replace content of a wiki article with new content
+        """
+        edit_button = self.q(css='.fa-pencil')
+        edit_button.click()
```

Comment:
You are absolutely right about that. I spun my wheels a bit on this, and because of the whole page object model, we can't really do a wait_for here because we're landing on the new page. (I can't say I like that pattern :( )

So what I did is over here, with the wait_for_page, which basically wraps a wait_for around the <page object>.is_browser_on_page.  https://github.com/edx/edx-platform/pull/7439/files#diff-b4e80030a775a77dbd123ef6635888b8R470